### ISSUE TITLE
IOS functionality is great and trouble free but Android is poor.  This is bad for physical web uptake!

### ISSUE THREAD

**Issue by craigahunter** [google/physical-web#915]
After several issues myself and reading lots of feedback from users about their experiences with the physical web, I think it is important that peoples voices are heard and solutions are found to make the basic principles of the service a lot more "dependable" on Android devices.  Perhaps a dedicated forum to enable a serious discussion about how these issues are to be handled and what steps forward need to be taken.
I use an IOS device using Google chrome for PW notifications and have not had a single problem with it since I first enabled the notifications.  Android however is a different story!  Problems myself, and problems demonstrating to others setting up their devices.  Their android phones did not work during one meeting yesterday, when I was passed an IPAD to set up it worked straight away. (slightly embarrassing when I am trying to promote the Physical Web and its simplicity to other businesses)

One of the reasons for the Physical Web is to help create a simpler way for people to access the world around them, but my experience so far is that people will simply not get on board if it is a nightmare to setup in the first place.  It just won't happen, and I can see the whole project failing, which is a shame when people are investing lots of time and money on business ideas that should help the physical web gain momentum.  It is these businesses that will in the long term pay towards marketing costs and general promotions of the physical web.  If IOS can work seamlessly then why can android not be setup to do the same?  Is there a way the a constructive forum can be setup that allows us to provide business input as opposed to just technical input.  As a few users have suggested, a more relaxed approach needs to be taken until it gets momentum enough to start filtering notifications by popularity, and "dropping beacons off" etc etc.  Is getting to technical before its started!  Please Mr Jenson help us to make this happen so we can start to get this project out there to businesses and users.
Thanks :)

**Comment by oninross** [comment]
Hi @craigahunter, I agree with you on this one.  But you have to consider that different Android phones handle PW differently compared to Apple, which is just one device/OS that makes is easy to make it consistent.  Some Android devices do not have Nearby or Physical Web detection even though they have Chrome installed on their phones.  

Hi @scottjenson, I would like to add that by passing the URL through Google URL shortener, sometimes or should I say most of the time the endpoint url doesnt get detected by the mobile device.  The moment I switched from Google URL shortener to BitLy, the notification has been consistently showing

**Comment by cgslaughter** [comment]
I agree with @craigahunter that something needs to be done about documentation and communication, and how things are suppose to work on different Android versions, platforms(iOS, Android), and what the future of the project might bring. I myself have invested a lot of time into adding this tech to our offerings, only to see all of our beacons stop working about a month ago. (Sure if we pull up a special app we see them.)Every test I can run to figure out why passes. Still no answers, and a lot of people wasting a lot of money. At this point in time for any real public uses of beacons on Android is dead, with no real answer as to why.

If the project is going to ever get anywhere, better management and communication to the community that will help it get off the ground is needed. If not, just hang the R.I.P. sign now.

**Comment by dougdot3** [comment]
I'd agree with @cgslaughter assessment. 

We're running a project in locations with close to 100k foot traffic per day and have never seen more than a very very small handful of interactions. The ROI is completely unsustainable but moreso, the inability to even assess reasons or do a 'deep dive' into how it's supposed to work vs how it's actually working makes it impossible to adapt, adjust or to try to plan for the different contingencies and end user experiences.

Lack of documentation, benchmarks, or any sort of data that might help adjust messages makes it nearly impossible to make a better experience for the experience and we've seen no evidence that the existing system is actually "toggling" or testing the existing messaging.

**Comment by craigahunter** [comment]
In regards to physical eb uptake then I believe a natural momentum will take place at some point. When looking a deployment then supporting marketing material is essential to

1. Inform people that the physical web is available in that particular zone or area, and
2. To give informative advice as to its use and how to activate it.

I must admit it is working great on my android device at present but in order to get it to do this I have had to update this, update that, install this, then check this and if I'm lucky hey presto the physical web...  IOS on the other hand I just had to enable it notifications and it has never failed once in probably a year since I installed it.

My opinion is that if IOS can make it a simple and easy almost frictionless experience then android should find a way to do the same.  One of the key points he Google team make in respect of the physical web is that it removes the "friction" involved in the APP process.  At this point at least the App process is simple to do.  As I explained I tried to demonstrate the physical web to 2 people a once and I couldn't get any of there devices to work (Android) unless I downloaded an phyweb Scanner app.
@scottjenson and his team could perhaps roll out a simple version that is simple and easy for user uptake and enable it to grow organically and then introduce more stringent searching capabilities or perhaps release some realistic proposals about exactly where the physical web is going, future commitments and whether this is a project that is being taken seriously.  So please please can we have some sort of report of official statement about now and the future of the physical web.  Thanks
